### DISCUSSION TITLE
Channel connection info

### DISCUSSION THREAD

**Discussion by danh65** [beyondcode/laravel-websockets#1120] [Q&A]
Is there a way to communicate with the websockets server from a Laravel app and check if a given socket id is open or not? I looked at the "getChannels" and "getChannelInfo" methods, but they don't give me specific connection information.
Thanks, Dan

**Comment by LAGGOUNE-Walid** [comment]
Greetings! I believe there isn't a straightforward approach to achieve that. My suggestion would be to create your own websocket handler and synchronize socket IDs in a storage location such as Redis or MySQL. This way, you can check if a socket ID is connected by verifying its existence on db or redis , ext ...
